.readthedocs.yml

```diff
@@ -13,14 +13,24 @@ build:
   os: ubuntu-24.04
   tools:
     python: '3.12'
+
   apt_packages:
   - graphviz
+
   jobs:
+
+    create_environment:
+    - asdf plugin add uv
+    - asdf install uv latest
+    - asdf global uv latest
+    - uv venv --python=3.12
+
+    install:
+    - uv sync --extra docs --frozen --exact
+
+    build:
+      html:
+      - uv run --extra docs --frozen --exact nox -s docs
```

Comment:
Using `--exact` ensures that the documentation builds are as consistent as possible across platforms, with the additional benefit that we'll likely be able to find problems with `uv.lock` sooner rather than later. The disadvantage is that the RTD builds _might_ become more finicky.